src/erllambda_poller.erl

```diff
@@ -88,12 +88,14 @@ init([]) ->
     Handler = handler_module(),
     erllambda:print_env(),
     erllambda:message("initializing ~p for handler ~p", [?MODULE, Handler]),
-    {ok, #state{
-          runtime_addr = Addr,
-          handler = Handler,
-          %% give it some time to start off
-          timer_ref = erlang:send_after(100, self(), poll)
-    }}.
+    State = #state{runtime_addr = Addr, handler = Handler},
+    case init:notify_when_started(self()) of
+        started ->
+            erllambda:message("Init had already started"),
+            {ok, State#state{timer_ref = erlang:send_after(0, self(), poll)}};
```

Comment:
aside from the fact that `init:notify_when_started` is kinda internal, would not this send create a second timer running?  given that will at  https://github.com/alertlogic/erllambda/pull/65/files#diff-58a77c2d3d9555a023a3e4c7ee0ef1e2R116

Reply:
That's one of the solutions I can think of that does not require changes in lambdas.
Yes, it is internal. It's not been changed since... forever.
> would not this send create a second timer running?
Would you please elaborate, sorry, not sure I got this.
`init:notify_when_started/1` does not send a new message and replies with `started` if init is in `started` state, otherwise it accepts a request and sends `{init, started}` message when init is started. This change would schedule `poll` event when either `started` message is received or init is already in `started` state.

Reply:
okay....but does it really protect? it waits for the erllambda server to start and not the top level module one.

Reply:
Yes, it does, unless I missed something.
Notifications [are sent](https://github.com/erlang/otp/blob/4d72eedc0d3573e9709b8f2c8773429e85529d7f/erts/preloaded/src/init.erl#L340) to all [subscribers](https://github.com/erlang/otp/blob/4d72eedc0d3573e9709b8f2c8773429e85529d7f/erts/preloaded/src/init.erl#L475) after when `{progress, started}` step from boot script [is evaluated](https://github.com/erlang/otp/blob/4d72eedc0d3573e9709b8f2c8773429e85529d7f/erts/preloaded/src/init.erl#L338) which is the last step in a boot process:

``` erlang
{script,
    {"erllambda_sam_example","1.2.0"},
    [{preLoaded,
         [erl_prim_loader,erl_tracer,erlang,erts_code_purger,
          erts_dirty_process_code_checker,erts_internal,
          erts_literal_area_collector,init,otp_ring0,prim_eval,prim_file,
          prim_inet,prim_zip,zlib]},
     {progress,preloaded},
     %% .. skipped for brevity ...
     {progress,kernel_load_completed},
     %% .. skipped for brevity ...
     {progress,modules_loaded},
     %% .. skipped for brevity ...
     {progress,init_kernel_started},
     %% .. skipped for brevity ...
     {progress,applications_loaded},
     %% .. skipped for brevity ...
     {apply,{c,erlangrc,[]}},
     {progress,started}]}.
```